### ISSUE TITLE
Scrollbar restored to wrong location

### ISSUE THREAD

**Issue by paulewert** [lmartinking/fix-sharepoint-scrolling#3]
The scrollbar will be restored, but it is put in the wrong location. The scroll bar should not scroll the nav bar and ribbon off the page. Makes it very difficult to use these elements on long pages.

**Comment by lmartinking** [comment]
Hi Paul, Thanks for the feedback! Do you know what version of Sharepoint it is?

**Comment by paulewert** [comment]
It was SharePoint 2010. I have re-tested, and your solution does keep the scrollbar in the correct location. However, it wasn't quite the same test as when I reported the issue: our environment is giving the scrollbar just fine normally now (i.e. without your changes). I believe our issue may have been different from the one you wrote this Chrome extension for but manifested itself in a similar manner. In our case, we found we had not only lost the scrollbar in Chrome (the first thing we noticed), but the Site Actions drop-down didn't work and other browsers were also affected (especially Safari--IE continued to work). We narrowed our issue down to a conflict with an Adobe Analytics script being added to the pages. Under certain conditions (what appeared to be a race), the Adobe script blocked SharePoint from executing it's own queue of functions to execute on page load. Those functions help with page re-sizing, Site Actions drop-down, scrolling, and several other things.
